File: banqueEJB/comptedepot/Models/CompteDepot.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace comptedepot.Models
{
    [Table("compte_depot")]
    public class CompteDepot
    {
        [Column("id")]
        public int Id { get; set; }
        [Column("client_id")]
        public int ClientId { get; set; }
        [Column("solde")]
        public decimal Solde { get; set; }
        [Column("taux_interet")]
        public decimal? TauxInteret { get; set; }
        [Column("plafond_retrait")]
        public decimal? PlafondRetrait { get; set; }
        [Column("date_dernier_interet")]
        public DateTime DateDernierInteret { get; set; }
        [Column("actif")]
        public bool Actif { get; set; }
        // public Client Client { get; set; }
    }
}

File: banqueEJB/centralisateur/src/main/java/com/centralisateur/depot/CompteDepot.java
package com.centralisateur.depot;

public class CompteDepot {
    private int id;
    private int clientId;
    private double solde;
    private Double tauxInteret;
    private Double plafondRetrait;
    private String dateDernierInteret;
    private boolean actif;

    // Getters et setters
    public int getId() { return id; }
    public void setId(int id) { this.id = id; }
    public int getClientId() { return clientId; }
    public void setClientId(int clientId) { this.clientId = clientId; }
    public double getSolde() { return solde; }
    public void setSolde(double solde) { this.solde = solde; }
    public Double getTauxInteret() { return tauxInteret; }
    public void setTauxInteret(Double tauxInteret) { this.tauxInteret = tauxInteret; }
    public Double getPlafondRetrait() { return plafondRetrait; }
    public void setPlafondRetrait(Double plafondRetrait) { this.plafondRetrait = plafondRetrait; }
    public String getDateDernierInteret() { return dateDernierInteret; }
    public void setDateDernierInteret(String dateDernierInteret) { this.dateDernierInteret = dateDernierInteret; }
    public boolean isActif() { return actif; }
    public void setActif(boolean actif) { this.actif = actif; }
}